lib/kitchen/patches/array.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+# Monkey patches for +Array+
+#
+class Array
+
+  # Receives a string to add as prefix in each item of the array
```

Comment:
Make it clear that this is returning a new array (not modifying this array in place)